apps/web/cypress/tests/digest-playground.spec.ts

```diff
@@ -32,4 +32,103 @@ describe('Digest Playground Workflow Page', function () {
 
     cy.url().should('include', '/templates/edit');
   });
+
+  it('should show the digest workflow hints', () => {
```

Comment:
the e2e test that verifies the whole flow:
--> from the templates page --> digest playground --> workflow editor --> all digest workflow hints